using_images/other_images/jenkins.adoc

```diff
@@ -214,17 +214,24 @@ slave.
 For more details on the sample job included in this image, see this link:https://github.com/openshift/origin/blob/master/examples/jenkins/README.md[tutorial].
 
 ifdef::openshift-origin[]
-== OpenShift Pipeline Plug-in
-
-The Jenkins image's list of pre-installed plug-ins includes a plug-in which assists in the creating of CI/CD workflows that run against
-an {product-title} server.  A series of build steps, post-build actions, as well as SCM-style polling are provided which equate to administrative
-and operational actions on the {product-title} server and the API artifacts hosted there.
-
-In addition to being accessible from the classic "freestyle" form of Jenkins job, the build steps as of version 1.0.14 of the OpenShift Pipeline Plug-in
-are also avaible to Jenkins Pipeline jobs via the DSL extension points provided by the Jenkins Pipeline Plug-in.
-
-The https://github.com/openshift/jenkins/tree/master/1/contrib/openshift/configuration/jobs/OpenShift%20Sample[sample Jenkins job] that is pre-configured in the Jenkins image utilizes the OpenShift pipeline plug-in and serves as an example of
-how to leverage the plug-in for creating CI/CD flows for {product-title} in Jenkins.
+== {product-title} Pipeline Plug-in
+
+The Jenkins image's list of pre-installed plug-ins includes a plug-in which
+assists in the creating of CI/CD workflows that run against an {product-title}
+server.  A series of build steps, post-build actions, as well as SCM-style
+polling are provided which equate to administrative and operational actions on
+the {product-title} server and the API artifacts hosted there.
+
+In addition to being accessible from the classic "freestyle" form of Jenkins
+job, the build steps as of version 1.0.14 of the {product-title} Pipeline
+Plug-in are also avaible to Jenkins Pipeline jobs via the DSL extension points
+provided by the Jenkins Pipeline Plug-in.
+
+The
+https://github.com/openshift/jenkins/tree/master/1/contrib/openshift/configuration/jobs/OpenShift%20Sample[sample
+Jenkins job] that is pre-configured in the Jenkins image utilizes the OpenShift
```

Comment:
I think ya missed one here "Jenkins image utilizes the OpenShift pipeline plug-in"?